examples/pypi/run.sh

```diff
@@ -0,0 +1,22 @@
+set -x
+set -e
+
+# Start artipie.
+docker run --name artipie -d -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Build and upload python project to Artipie
```

Comment:
Please, end the sentence with a period for the sake of consistency with other comments.

Reply:
Done, also made first letter not capital as it's written in other sentences.